### PR TITLE
:ambulance: Changes to enable arm64 builds

### PR DESCRIPTION
**Pull request by fabriceb**
Based on @chris-araman's current pull request. I added to this the removal of all mentions of VALID_ARCH which blocks the build on arm64 architectures.

### CODE REVIEW

**Comment by chris-araman** [comment]
@fabriceb, instead of modifying committed sources here, I suspect @phatblat is going to want to take these as a new pull request for upstream [Commandant](https://github.com/Carthage/Commandant) and an update to [Nimble 9](https://github.com/Quick/Nimble/releases/tag/v9.0.0), which already supports Apple Silicon.

**Comment by chris-araman** [comment]
Upstream PR here:
https://github.com/Carthage/Commandant/pull/158

File: script/install [hunk 1]
```diff
@@ -27,10 +27,6 @@ INSTALL_TEMPORARY_FOLDER=${DSTROOT:-build/distributions}
 # Final destination.
 PREFIX=/usr/local
```
Comment (Cyberbeni) [inline script/install:28]: this will be /opt/homebrew on ARM64

Comment (Cyberbeni) [reply script/install:28]: It is already being passed in the formula. But having a default that only works on x64 might confuse some people why they can't manually install it if they do some modifications and want to use it before it gets merged/released.

**Comment by phatblat** [comment]
Yeah, @fabriceb I have mixed feelings about checking in the Carthage/Checkouts folder, but it's meant to just be a copy of the upstream dependencies. If VALID_ARCHS needs to be removed or updated in those projects, would you be open to submitting a PR to Quick, Nimble and/or Commandant?

@chris-araman are the install script and mas project changes already included in #310?

**Comment by chris-araman** [comment]
> @chris-araman are the install script and mas project changes already included in #310?

Yes. It's here:
https://github.com/mas-cli/mas/pull/310/commits/e8cdd18748a15f048dce55b798a3cde06dd3fdf5

**Comment by phatblat** [comment]
Cool, thanks for checking.

**Pull request closed without merging**